lib/oembed/providers/spotify.rb

```diff
@@ -0,0 +1,19 @@
+module OEmbed
+  class Providers
+    # Provider for spotify.com
+    # https://developer.spotify.com/documentation/embeds/reference/oembed
+    # https://developer.spotify.com/documentation/embeds/tutorials/using-the-oembed-api
+    Spotify = OEmbed::Provider.new(
+      "https://open.spotify.com/oembed",
```

Comment:
**Annotation:** It turns out that over the past few years Spotify has now officially documented their oEmbed API **and** has changed the URL they prefer for the API endpoint.